webapp/src/main/java/com/box/l10n/mojito/utils/NamingTransformation.java

```diff
@@ -0,0 +1,70 @@
+package com.box.l10n.mojito.utils;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import io.micrometer.core.instrument.util.StringUtils;
+
+public enum NamingTransformation {
+
+    NONE(Function.identity()),
+
+    CAMEL_CASE(s -> {
+        String[] parts = s.split("\\.");
+        StringBuilder conventionName = new StringBuilder(s.length());
+        for (int i = 0; i < parts.length; i++) {
```

Comment:
I've used it for the case conversion on line 29. It can't be applied globally as Guava's `CaseFormat` doesn't have the input format (`some.metric.name`) specified in the enum